java/client/src/org/openqa/selenium/remote/ProtocolHandshake.java

```diff
@@ -77,13 +77,13 @@ public Result createSession(HttpClient client, Command command)
       parameters = new StringBuilder("{");
       amendW3CParameters(parameters, des, req);
       parameters.append("}");
-      LOG.info("Falling back to straight W3C remote end connection");
+      LOG.fine("Falling back to straight W3C remote end connection");
       result = createSession(client, parameters);
     }
 
     if (result.isPresent()) {
       Result toReturn = result.get();
-      LOG.info(String.format("Detected dialect: %s", toReturn.dialect));
+      LOG.fine(String.format("Detected dialect: %s", toReturn.dialect));
```

Comment:
can we leave this one as info?

One has the ability to set java log levels... although it seemingly is too complicated for many users. But those that log issues this is pretty important information to see by default in the console when we potentially need to troubleshoot things.